### PR TITLE
social media info handbook section - initial

### PR DESCRIPTION
**Pull request by kierisi**
this is the updated section that contains all of our social media information. note that outward communication (blog, discourse, youtube) and zenodo are considered separate sections, although everything falls under the header of "Community Outreach and Communication", as follows:

# Community Outreach and Communication
* Linktree
* Blog
* Discourse
* YouTube
* Zenodo
* Social media platforms
** Social media accounts
** Content types (ad hoc v. evergreen)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207232741135324

### CODE REVIEW

**Comment by kierisi** [comment]
pre-commit.ci autofix

**Comment by lwasser** [comment]
@kierisi to fix the CI error you can add code 999 to  this line in our github action. this error is because linked in likes people to login. 

https://github.com/pyOpenSci/handbook/blob/main/.github/workflows/build-docs.yml#L55

File: community/social.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
-# pyOpenSci Social Media
+# pyOpenSci Social Media Platforms
+The 2024 North Star for pyOpenSci’s social media accounts is to raise general awareness of pyOpenSci in the broader open source community through both scheduled and ad hoc content that reflects our commitments to diversity and inclusion within the open source scientific Python community.
```
Comment (lwasser) [inline community/social.md:2]: Perhaps also add - to build a contributor / volunteer community that supports our mission of empowering scientists to create better software that supports more open and reproducible science

**Review by lwasser — APPROVED** [review]
@kierisi let's merge this. you can either add that 999 code in a separate pr or add it here. i added one note about the north start for social. i do think it's about our DEIa mission. but it's also about supporting scientists both in creating software and making their science more sharable and open too.

**Comment by kierisi** [comment]
pre-commit.ci autofix

**Pull request merged**